### ISSUE TITLE
Update to MPFIT v1.5?

### ISSUE THREAD

**Issue by mbaz** [gcalderone/CMPFit.jl#5]
I just noticed that verstion 1.5 of the MPFIT library was released back in December 2022.

**Comment by gcalderone** [comment]
Yes, it is in my TODO list.  Hope to do it in the next ~few weeks.
